docs/podman-images.1.md

```diff
@@ -23,6 +23,26 @@ Show image digests
 
 Filter output based on conditions provided
 
+  Filters:
+
+  **after**
+    Filter on images created after the given time.Time
+
+  **before**
+    Filter on images created before the given time.Time
+
+  **dangling=true**
+    Show dangling images.  Dangling images are layers that have no relationship to any tagged images. They do not serve a purpose and consume disk space.
```

Comment:
Suggest:
"Show dangling images.   Dangling images are a file system layer that was used in a previous build of an image and are no longer referenced by any active images.  They are denoted with the `<none>` tag, consume disk  space and server no active purpose."

Reply:
just updated my previous comment to fix some pre-tea grammar

Reply:
Nice job, I like your description.